tox.ini

```diff
@@ -33,11 +33,9 @@ passenv = {[testenv]passenv}
           # without PROGRAMDATA cloning using git for Windows will fail with an
           # `error setting certificate verify locations` error
           PROGRAMDATA
-extras = lint
+extras = lint, testing, docs, publish
```

Comment:
I think you only need `lint` here

Reply:
needed all these I believe to seed isort, testing to have pytest, docs to have sphinx, publish for stuff used in setup.py

Reply:
`seed-isort-config` runs in an isolated environment -- it doesn't actually need any dependencies :D

Reply:
how does it then differentiate between local/thirdparty/stdlib?

Reply:
`aspy.refactor_imports` is clever: https://github.com/asottile/aspy.refactor_imports/blob/9acde162c907d0e28b665dd906ac032052ba300c/aspy/refactor_imports/classify.py#L121-L142

tl;dr:

1. try to use the import machinery (it'll probably fail), if it succeeds classify based on whether it is from a `X-packages` directory => (system or third party)
2. look on the local filesystem for the first import segment => (first party)
3. otherwise => (third party)

Reply:
sounds great